cluster/saltbase/salt/calico/master.sls

```diff
@@ -0,0 +1,70 @@
+{% if pillar.get('policy_provider', '').lower() == 'calico' %}
+
+calicoctl:
+  file.managed:
+    - name: /usr/bin/calicoctl
+    - source: https://github.com/projectcalico/calico-docker/releases/download/v0.18.0/calicoctl
+    - source_hash: sha256=694d2e0d0079980d75bc807fcb6626b18e5994638aa62743d45b906e742a0eed
+    - makedirs: True
+    - mode: 744
+
+calico-policy:
+  file.managed:
+    - name: /usr/bin/policy
+    - source: https://github.com/projectcalico/k8s-policy/releases/download/v0.1.3/policy
+    - source_hash: sha256=def1b53ec0bf3ec2dce9edb7b4252a514ccd6b06c7e738a324e0a3e9ecf12bbe
+    - makedirs: True
+    - mode: 744
+
+calico-etcd:
+  cmd.run:
+    - unless: docker ps | grep calico-etcd
+    - name: >
+               docker run --name calico-etcd -d --restart=always -p 6666:6666
+               -v /varetcd:/var/etcd
+               gcr.io/google_containers/etcd:2.2.1
```

Comment:
I think it's best to stick with the latest version that's already used in the Kubernetes code base, rather than getting a newer one from an alternate source.  We have no particular dependency anyway.